tensorflow/tsl/util/BUILD

```diff
@@ -26,9 +26,19 @@ package(
     licenses = ["notice"],
 )
 
+filegroup(
+    name = "mobile_srcs_byteswaparray",
```

Comment:
Rename this to "mobile_srcs_no_runtime".

In `//tensorflow/core/util/BUILD`, under "mobile_srcs_no_runtime" add this to "srcs":
```
"//tensorflow/tsl/util:mobile_srcs_no_runtime",
```
Turns out this is the missing dependency for the broken android builds.

Reply:
Done. Thanks!